data/json/items/armor/robofac_armor.json

```diff
@@ -540,7 +540,7 @@
     "type": "ARMOR",
     "copy-from": "robofac_greaves",
     "name": { "str_sp": "Hub 01 prototype greaves" },
-    "description": "3D printed leg guards, despite its rough finish it's still lightweight and secure.  This is made of 3D printed plastic with an unhardened metal core to give some reinforcement.",
+    "description": "3D printed leg guards, consisting of a thick knee pads and a pair of large, flexible polymer plates for securing around the calves to protect the tibias of each leg.  Two straps on each plate can be tightened or loosened to fit different leg sizes.  Despite the rough finish, it doesn't impede your movement, fits comfortably against your body.",
```

Comment:
Maybe an "and" in the last sentence I assume that last comma is supposed to be an Oxford comma?